syft/execution/plan.py

```diff
@@ -211,14 +220,15 @@ def build(self, *args, trace_autograd=False):
         if self.include_state:
             args += (self.state,)
 
-        with trace(framework_packages["torch"], self.role, self.owner) as wrapped_torch:
```

Comment:
If the context manager is not used anymore, we can remove it from the imports.
And maybe remove it totally if it works without? What do you think @karlhigley?

Reply:
Sure, let’s get rid of anything we don’t need.